trino-api/config/default.toml

```diff
@@ -0,0 +1,18 @@
+[app]
+    appEnv              = "default"
+    serviceName         = "trino-client"
+    hostname            = "localhost"
+    port                = 8000
+    metricsPort         = 8080
+    shutdownTimeout     = 2
+    shutdownDelay       = 2
+    maxRecords          = 5000
+[db]
```

Comment:
make it a subsection of previous key and rename to 'trino-backend-db' or something